build.gradle

```diff
@@ -242,6 +242,17 @@ dependencyManagement {
         dependencySet(group: 'commons-beanutils', version: '1.9.4') {
             entry 'commons-beanutils'
         }
+
+        // solves CVE-2020-5407
+        dependencySet(
+                group: 'org.springframework.security',
+                version: '5.3.2.RELEASE'
```

Comment:
They have suppressed this on master, i was getting similar build problems to this one when i tried to use it

Reply:
removed